src/components/bookForm.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+function BookForm() {
+  return (
+    <form action="" className="" style={{ marginTop: '50px', marginBottom: '150px' }}>
+      <h4 className="pb-4">ADD NEW BOOK</h4>
+      <div className="d-flex w-100 justify-content-between">
+        <input
+          style={{ width: '50%' }}
+          type="text"
+          className="form-control py-3"
+          name=""
+          id=""
+          aria-describedby="helpId"
+          placeholder="Book title"
+        />
+        <input
+          style={{ width: '30%' }}
+          type="text"
+          className="form-control py-3"
+          name=""
+          id=""
+          aria-describedby="helpId"
+          placeholder="Category"
+        />
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you update this placeholder text to be `Author` as this input field will be used for `Author` and not for `Category`.